README.md

```diff
@@ -54,11 +54,11 @@ Of course you can also add a totally new exercise, but it might be a good idea t
 
 #### Updating an exercise test suite
 Updating a test suite of an existing exercise is special because it usually affects all languages.
-You can refer to the [documentation on updating a test suite](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#updating-an-exercise-test-suite).
+You can refer to the [documentation on updating a test suite](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md#updating-an-exercise-test-suite).
```

Comment:
Yes, this would be the link: https://github.com/exercism/docs/blob/master/language-tracks/exercises/anatomy/test-suites.md I'll change it.